client/src/client.js

```diff
@@ -86,7 +86,28 @@ function line_operations(a_line) {
  ***********************************************************/
 async function two_lines() {
   // calling our lambda function
-  var cr_call = await axios.get('https://91duv1eln6.execute-api.us-east-1.amazonaws.com/dev/chartdb');
+  //var cr_call;
```

Comment:
It is best practice to not leave commented code in your check-in. You can always use git history if you need to get something back.

Reply:
I got rid of all of the commented code